public/components/trace_analytics/components/dashboard/top_groups_page.tsx

```diff
@@ -29,12 +29,12 @@ export function TopGroupsPage(props: {
   const toggleButtons = [
     {
       id: 'error_rate',
-      label: 'Error rate',
+      label: 'Errors',
       'data-test-subj': 'errors-toggle'
     },
     {
       id: 'throughput',
-      label: 'Throughput',
+      label: 'Rate',
```

Comment:
Given the wider context that we're passing "Error Rate" and "Request Rate" around, does just "Rate" make clear sense here?

Reply:
Added the Request Rate here to make that more clear